scipy/special/cephes/hyp2f1.c

```diff
@@ -590,3 +601,23 @@ static double hyp2f1ra(double a, double b, double c, double x,
 
     return f0;
 }
+
+
+/*
+    15.4.2 Abramowitz & Stegun.
+*/
+static double hyp2f1_neg_c_equal_bc(double a, double b, double x)
+{
```

Comment:
This should be (untested):

```
double k;
double collector = 1;
double sum = 1;

for (k = 1; k <= -b; k++) {
    collector *= (a + k - 1)*x/k;
    sum += collector;
}
```

Reply:
Yeah, the problem was that I did not expand the `poch` function function so I did not expand it.